pt-BR/mail/chrome/messenger/credits.dtd

```diff
@@ -1,7 +1,7 @@
 <!ENTITY credit.title     "Créditos do &brandFullName;">
-<!ENTITY credit.leads     "Engineering Leads">
-<!ENTITY credit.core      "Core Development Posse">
-<!ENTITY credit.gecko     "Gecko Layout Engine">
+<!ENTITY credit.leads     "Engenheiros-chefes">
+<!ENTITY credit.core      "Grupo de Desenvolvimento Principal">
+<!ENTITY credit.gecko     "Engine de Layouts Gecko">
```

Comment:
I'm not sure I agree with this. I've never seen this use of Motor in PTBR. Could you show me a source?